src/work-allocation-2/containers/task-assignment-confirm/task-assignment-confirm.component.ts

```diff
@@ -29,6 +29,17 @@ export class TaskAssignmentConfirmComponent implements OnInit {
     private readonly messageService: InfoMessageCommService) {
   }
 
+  private get returnUrl(): string {
+    let url: string = '/work/my-work/list';
```

Comment:
this is still hardcoded, is that correct?

Reply:
@rayliangatos - that's correct. The default return URL is the "My work" screen and is used as a fall-back if there is no other return URL.

Reply:
I double check the code, actually this default value is useless, and it will be always overrided by:
if (window && window.history && window.history.state && window.history.state.returnUrl) { url = window.history.state.returnUrl; }

In that case could you assign the empty value to it? like:

Reply:
Similar to above, I will change the default return URL to `''`.